src/e2e-test/resources/BigQuery/BigQueryCreateTableQuery.txt

```diff
@@ -1,3 +1,4 @@
-create table `DATASET.TABLE_NAME` (Name STRING, Col_Timestamp__c TIMESTAMP, Col_Date__c DATE, Col_Currency__c FLOAT64,
- Col_Email__c STRING, Col_Number__c FLOAT64, Col_GeoLocation__Latitude__s FLOAT64,
-Col_GeoLocation__Longitude__s FLOAT64, Col__c STRING, Col_Url__c STRING, Col_Time__c TIME, Col_Text__c STRING)
+create table `DATASET.TABLE_NAME` (Id__c FLOAT64, Name STRING, Col_Timestamp__c TIMESTAMP, Col_Date__c DATE, Col_Currency__c FLOAT64,
```

Comment:
Is this change done to fix the broken environment dependent tests?

Reply:
Yes, we have done this change to fix the broken environment dependent tests.